evals/ai_api_calls/get_open_ai_completion.py

```diff
@@ -57,3 +60,22 @@ def get_code_block_from_message(message: str) -> str:
         return message
     
     return message.split('```python\n')[1].split('\n```')[0]
+
+def get_sql_from_message(message: str, model: str) -> SQLDetails:
+    """
+    Extract the first SQL query + other metadata from a message.
+    """
+    client = OpenAI(api_key=os.getenv("OPENAI_API_KEY"))
+    response = client.beta.chat.completions.parse(
+        model=model,
+        messages=[
+            {"role": "system", "content": "You are a helpful assistant that extracts SQL queries from messages."},
+            {"role": "user", "content": message}
+        ],
+        response_format=SQLDetails,
```

Comment:
Maybe just for a little bit more clarity we can call this `ParsedSQLDetails` or something like that.